account_voucher_killer/README.rst

```diff
@@ -1,8 +1,29 @@
-.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+=========================
+Accounting Payment Access
+=========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-
-Prevent the usage of payments from invoices
-===========================================
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fbank--payment-lightgray.png?logo=github
+    :target: https://github.com/OCA/bank-payment/tree/11.0/account_voucher_killer
+    :alt: OCA/bank-payment
```

Comment:
The repo referred in the PR in several places is still bank-payment instead of account-payment

Reply:
@luismontalba readme updated. Thanks!